src/controllers/issue.controller.ts

```diff
@@ -0,0 +1,42 @@
+import { Request, Response } from 'express';
+import { IssueRepository } from '../repositories/issue.repository';
+
+export class IssueController {
```

Comment:
In node.js the convention is that these are services. And the C# controllers are routers in node.js.
Also routers are handling the request and response and no the services.

Reply:
Rename this to `IssueService`